packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,43 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  background-color: var(--gcds-button-text-only-default-background);
+
+  &
+  :not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-button-text-only-default-decoration-thickness);
+  }
+
+  &
+  :hover {
+    background-color: var(--gcds-button-text-only-default-background);
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-button-text-only-hover-decoration-thickness);
+  }
+
+  &
+  :focus {
+    background-color: var(--gcds-button-shared-focus-background);
```

Comment:
Can you use the tokens you created in https://github.com/cds-snc/gcds-tokens/pull/173?

Reply:
Applies to lines 24-29